docs/index.rst

```diff
@@ -7,6 +7,39 @@
 Welcome to bodhi's documentation!
 =================================
 
+What is Bodhi?
+--------------
+Bodhi is a modular web-based system that facilitates the process of publishing package updates for Fedora. Bodhi is the application that manages updates for Fedora releases. It maintains a single stage of repositories by adding/updating/removing packages. It can be used to create or modify updates and overrides.
```

Comment:
When you say that Bodhi is modular, what do you mean? I haven't thought of it as being modular before.

The first and second sentence seem a little redundant.

What does it mean that Bodhi maintains a single stage of repositories?

I think we can go for something a little simpler here. Perhaps something like "Bodhi is designed to democratize the package update testing and release process for RPM based Linux distributions. It provides an interface for developers to propose updates to a distribution, and an interface for testers to leave feedback about updates through a +1/-1 karma system."

Also, let's try to keep line lengths no longer than 100 columns.